.flake8

```diff
@@ -1,3 +1,3 @@
 [flake8]
 max-line-length = 110
-ignore = E731
+ignore = E731,W504
```

Comment:
Using the latest flake8, the W503 (line break before binary operator) and W504 (line break after binary operator) warnings are both enabled by default, which probably shouldn't be the case, since those two warnings are mutually exclusive: http://pycodestyle.pycqa.org/en/latest/intro.html#error-codes

> The check W503 is mutually exclusive with check W504

So we have to ignore at least one, or there's no valid way to break a line around a binary operator.

Reply:
I see. thanks for the info.